cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -235,6 +235,11 @@ protected ClientConfig getClientConfig() {
       builder.setAccessToken(accessToken);
     }
 
+    String verifySSL = System.getProperty("verifySSL");
+    if (verifySSL != null) {
+      builder.setVerifySSLCert(Boolean.valueOf(verifySSL));
+    }
```

Comment:
I think the default should be consistent with the default in the ClientConfig, so that there aren't two places where a default is defined.